js/adapt-flipper.js

```diff
@@ -54,6 +55,10 @@ define(function(require) {
             }, this);
         },
 
+        setScreenReaderVisibility: function() {
+            this.$('.flipper-active-item').empty().prepend(this.$(".flipper-item.state-1").clone());
```

Comment:
Use single quotes at end

Reply:
Rest of file uses double quotes so I will change this to use double